app-creator/data/templates/templates.go

```diff
@@ -0,0 +1,50 @@
+package templates_data 
+
+import (
+	"encoding/json"
+	"io"
+	"context"
+	"cloud.google.com/go/datastore"
+	"log"
+)
+
+type Template struct {
+	Id string `json:"id"` 
+	Name string `json:"name"`
+	ImageUrl string `json:"imageUrl"`
+	Template string `json:"template"`
+}
+
+type Templates []*Template
+
+/**
+* Converts an array of Template data into JSON.
```

Comment:
Doc comments should begin with the documented name (https://golang.org/doc/effective_go.html#commentary)